src/sentry/static/sentry/app/actionCreators/sentryAppTokens.jsx

```diff
@@ -0,0 +1,51 @@
+import {
+  addErrorMessage,
+  addLoadingMessage,
+  addSuccessMessage,
+  clearIndicators,
+} from 'app/actionCreators/indicator';
+import {t} from 'app/locale';
+
+/**
+ * Install a sentry application
+ *
+ * @param {Object} client ApiClient
+ * @param {Object} app SentryApp
+ */
+export async function addSentryAppToken(client, app) {
+  addLoadingMessage();
+  try {
+    const resp = await client.requestPromise(`/sentry-apps/${app.slug}/api-tokens/`, {
```

Comment:
Do you still use `requestPromise` with `await` or is there a different method for that?

Reply:
@mnoble `requestPromise` returns a promise so yea, using `await` makes sense.